src/app.module.ts

```diff
@@ -1,9 +1,35 @@
 import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { ConfigModule, ConfigService } from '@nestjs/config';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { EnvConfiguration } from './config/env.config';
```

Comment:
You can use path alias, A Path Aliases is a way to rename a path to make it easier to use.
ref: https://christianlydemann.com/simpler-typescript-paths-with-path-aliases/

Reply:
Great!